views/group/index.pug

```diff
@@ -1,9 +1,20 @@
 extends ../layouts/main
 
+include ./pagination/activeButton.pug
+include ./pagination/inactiveButton.pug
+
 block content
   div(class='flex flex-row items-center space-x-4')
     h1(class='title') Csoportok
     a(href='/groups/new' class='text-xl btn btn-primary animate-hover') Új csoport
+  div(class='w-1/2 mx-auto text-xl font-bold sm:text-lg')
+    div(class='my-5 flex flex-row items-center justify-center space-x-2')
+      if past === 'true'
+        +inactiveButton('/groups', 'Következők')
+        +activeButton('/groups?past=true', 'Korábbiak')
+      else
+        +activeButton('/groups', 'Következők')
+        +inactiveButton('/groups?past=true', 'Korábbiak')
```

Comment:
Maybe the text Következők is not UX friendly enough. What do you think of 'Elkövetkezőek' or 'Indulás előttiek'?

Reply:
Facebook uses 'Közelgő események', but all the options sounds weird, especially without 'események' or 'csoportok' at the end, but otherwise it would be quite long... (especially on mobile)
![image](https://user-images.githubusercontent.com/13004605/127868824-e0b3b2a9-14e7-481c-990a-414c2ebccd6a.png)
![image](https://user-images.githubusercontent.com/13004605/127868847-cbb3d7b8-bf12-47cb-a20c-28e48c59397e.png)

Reply:
Yeah true, I think this can stay as 'Következők' :)